lesson 16/index.js

```diff
@@ -0,0 +1,92 @@
+"use strict"
+//task 1
+let x = 20;
+let y = 58;
+let z = 42;
+let summ = x + y + z;
+console.log (summ);
+
+//task 2
+let year = prompt ('Введи колличество лет');
+let days = year * 365;
+let hours = days * 24;
+let minutes = hours * 60;
+let seconds = minutes * 60;
+console.log (days);
+console.log (hours);
+console.log (minutes);
+console.log (seconds);
+let myAge = prompt ('Сколько мне лет');
+let myAgeInSeconds = myAge * 365 * 24 * 60 * 60;
+console.log (myAgeInSeconds);
+
+//task 3
+let count = 42;
+let userName = '42';
+userName = Number (userName);
+console.log (typeof userName);
+userName = +userName;
+console.log (typeof userName);
+count = String (count);
+console.log (typeof count);
+count = '42';
+console.log (typeof count);
+
+//task 4
+let a = 1;
+let b = 2;
+b = String (b);
+a = String (a);
+let c = "белых медведей";
+let d = a + b + ' ' + c;
+console.log (d);
+//task 5
+let w1 = "доступ";
+let w2 = "морпех";
+let w3 = "наледь";
+let w4 = "попрек";
+let w5 = "рубило";
+let l1 = w1.length;
+let l2 = w2.length;
+let l3 = w3.length;
+let l4 = w4.length;
+let l5 = w5.length;
+let lengthWords = l1 + l2 + l3 + l4 + l5;
```

Comment:
Еще как вариант можно сделать это короче:
```suggestion
const wordsLength = (w1 + w2 + w3 + w4 + w5).length;
```